### ISSUE TITLE
Add logits support to discrete distributions

### ISSUE THREAD

**Issue by neerajprad** [pyro-ppl/numpyro#51]
Since we use float32 values by default instead of float64 like `scipy.stats` does, having support for logits in multinomial, bernoulli and binomial distributions is important, and will make  operations that manipulate probabilities more numerically stable.
